ChatIntegrationExamples/README.md

```diff
@@ -147,7 +150,45 @@ To send a notification in a channel:
    - *Channel ID*: The ID of the channel where you want to send a notification. Note that this must be a channel of the specified team.
    - *Notification*: The text of the notification.
 
-4. Click the '*execute now*' button. A notification will be sent in Microsoft Teams. <details><summary>`show demo`</summary>![Gif-SendNotification](https://user-images.githubusercontent.com/109528797/187139103-4728e148-204d-447f-9674-8d74f4e373d1.gif)</details>
+4. Click the '*execute now*' button. A notification will be sent in the channel in Microsoft Teams. <details><summary>`show demo`</summary>![Gif-SendChannelNotification](https://user-images.githubusercontent.com/109528797/187139103-4728e148-204d-447f-9674-8d74f4e373d1.gif)</details>
+
+#### Creating a Private Chat
+
+1. In the Automation module in DataMiner Cube, click the '*Create Private Chat Example*' Automation script and click the '*Execute*' button.
```

Comment:
```suggestion
1. In the Automation module of DataMiner Cube, click the '*Create Private Chat Example*' Automation script and click the '*Execute*' button.
```

Reply:
Automation an also be used outside cube so 'in' is better in this case